### ISSUE TITLE
[BUG] cannot enable graylog-server on openSUSE 15

### ISSUE THREAD

**Issue by drewmiranda-gl** [Graylog2/fpm-recipes#138]
Installing Graylog on openSUSE 15 (tested with 15.5) does not allow the user to enable the `graylog-server` service. They receive the following error:

```
localhost:~ # systemctl enable graylog-server
Synchronizing state of graylog-server.service with SysV service script with /usr/lib/systemd/systemd-sysv-install.
Executing: /usr/lib/systemd/systemd-sysv-install enable graylog-server
ln -sf ../graylog-server /etc/init.d/rc2.d/S50graylog-server
ln: failed to create symbolic link '/etc/init.d/rc2.d/S50graylog-server': No such file or directory
```

Datanode, however, does not have this issue and works as expected. This appears to be because the `recipe.rb` for `graylog-server` is using `init.d`, See https://github.com/Graylog2/fpm-recipes/blob/972b00cb221ea21348297a6e9bd30e73bbc088e0/recipes/graylog-server/recipe.rb#L68

While datanode is using `systemd`, See https://github.com/Graylog2/fpm-recipes/blob/972b00cb221ea21348297a6e9bd30e73bbc088e0/recipes/graylog-datanode/recipe.rb#L59

Upon further testing, I was able to successfully enable `graylog-server` if i deleted:

```
sudo rm /etc/init.d/graylog-server
```

```
localhost:/etc/init.d # systemctl enable graylog-server
Created symlink /etc/systemd/system/multi-user.target.wants/graylog-server.service → /usr/lib/systemd/system/graylog-server.service.
```

I believe removing lines 68/69 would resolve, https://github.com/Graylog2/fpm-recipes/blob/972b00cb221ea21348297a6e9bd30e73bbc088e0/recipes/graylog-server/recipe.rb#L68

**Comment by bernd** [comment]
@drewmiranda-gl The server package installs `init.d`, `upstart`, and `systemd` configurations to maximize compatibility.

But I think we can safely remove the `init.d` and `upstart` configurations since every recent Linux distribution we support is now using systemd.

**Comment by bernd** [comment]
Fixed in #140.
